### ISSUE TITLE
Unable to Build and Debug Android

### ISSUE THREAD

**Issue by alfkonee** [dotnet/maui#12549]
### Description

When I build the Andriod App I get this Error
```
    Error MAUI0000: System.IO.FileNotFoundException: Could not load file or assembly 'System.Text.Encodings.Web, Version=7.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' or one of its dependencies. The system cannot find the file specified.
File name: 'System.Text.Encodings.Web, Version=7.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'
   at System.Text.Json.Nodes.JsonNode.ToJsonString(JsonSerializerOptions options)
   at Microsoft.Maui.Resizetizer.AppleIconAssetsGenerator.Generate() in D:\a\_work\1\s\src\SingleProject\Resizetizer\src\AppleIconAssetsGenerator.cs:line 87
   at Microsoft.Maui.Resizetizer.ResizetizeImages.ProcessAppIcon(ResizeImageInfo img, ConcurrentBag`1 resizedImages) in D:\a\_work\1\s\src\SingleProject\Resizetizer\src\ResizetizeImages.cs:line 147
   at Microsoft.Maui.Resizetizer.ResizetizeImages.<>c__DisplayClass26_0.<ExecuteAsync>b__0(ResizeImageInfo img) in D:\a\_work\1\s\src\SingleProject\Resizetizer\src\ResizetizeImages.cs:line 86
   at Microsoft.Maui.Resizetizer.AsyncTaskExtensions.<>c__DisplayClass0_0`1.<ParallelForEach>b__0(TSource s) in D:\a\_work\1\s\src\SingleProject\Resizetizer\src\AsyncTaskExtensions.cs:line 21

WRN: Assembly binding logging is turned OFF.
To enable assembly bind failure logging, set the registry value [HKLM\Software\Microsoft\Fusion!EnableLog] (DWORD) to 1.
Note: There is some performance penalty associated with assembly bind failure logging.
To turn this feature off, remove the registry value [HKLM\Software\Microsoft\Fusion!EnableLog].
 (532, 10)
```

### Steps to Reproduce

Create new Maui Project 
Debug the Project to Andriod 

### Link to public reproduction project repository

NA

### Version with bug

7.0 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

iOS, Android, I was *not* able test on other platforms

### Affected platform versions

VS 17.4.3

### Did you find any workaround?

None

### Relevant log output

```shell
Error MAUI0000: System.IO.FileNotFoundException: Could not load file or assembly 'System.Text.Encodings.Web, Version=7.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' or one of its dependencies. The system cannot find the file specified.
File name: 'System.Text.Encodings.Web, Version=7.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'
   at System.Text.Json.Nodes.JsonNode.ToJsonString(JsonSerializerOptions options)
   at Microsoft.Maui.Resizetizer.AppleIconAssetsGenerator.Generate() in D:\a\_work\1\s\src\SingleProject\Resizetizer\src\AppleIconAssetsGenerator.cs:line 87
   at Microsoft.Maui.Resizetizer.ResizetizeImages.ProcessAppIcon(ResizeImageInfo img, ConcurrentBag`1 resizedImages) in D:\a\_work\1\s\src\SingleProject\Resizetizer\src\ResizetizeImages.cs:line 147
   at Microsoft.Maui.Resizetizer.ResizetizeImages.<>c__DisplayClass26_0.<ExecuteAsync>b__0(ResizeImageInfo img) in D:\a\_work\1\s\src\SingleProject\Resizetizer\src\ResizetizeImages.cs:line 86
   at Microsoft.Maui.Resizetizer.AsyncTaskExtensions.<>c__DisplayClass0_0`1.<ParallelForEach>b__0(TSource s) in D:\a\_work\1\s\src\SingleProject\Resizetizer\src\AsyncTaskExtensions.cs:line 21

WRN: Assembly binding logging is turned OFF.
To enable assembly bind failure logging, set the registry value [HKLM\Software\Microsoft\Fusion!EnableLog] (DWORD) to 1.
Note: There is some performance penalty associated with assembly bind failure logging.
To turn this feature off, remove the registry value [HKLM\Software\Microsoft\Fusion!EnableLog].
 (532, 10)
```

**Comment by mattleibow** [comment]
This is interesting as the bump to use 7.0 of System.Text.* was here: (Dec 5, 2022) https://github.com/dotnet/maui/commit/a0dbf75ea1316e6cb5b945100cabe6425e3a9819
And then we reverted here: (Jan 6, 2023)  https://github.com/dotnet/maui/commit/b983ae74b74e22d73f108d914c36494f4261b9ae

This should not be in a public release.

Can you attach a binlog? https://github.com/xamarin/xamarin-macios/wiki/Diagnosis

**Comment by alfkonee** [comment]
[msbuild.zip](https://github.com/dotnet/maui/files/10395619/msbuild.zip)
Hello @mattleibow  I was finding it difficult installing the MAUI workloads to .NET 7  using the public repos only I had to add the eng repos as outlined here https://github.com/dotnet/sdk/issues/28613#issuecomment-1373381513 in order to get it to install 
That may also be another reason why the build s might be broken

**Comment by alfkonee** [comment]
The Problem seems to be fixed in the latest sdk 7.0.102

**Comment by rachelkang** [comment]
@alfkonee Can you confirm you are no longer seeing this issue and it be can be closed?
